src/engraving/libmscore/barline.cpp

```diff
@@ -570,7 +570,7 @@ bool BarLine::isBottom() const
     if (!_spanStaff) {
         return true;
     }
-    int idx = staffIdx();
+    size_t idx = staffIdx();
     if (idx == score()->nstaves() - 1) {
```

Comment:
Here `idx` could potentially go negative, if `nstaves()` is 0, or is that something not to consider?

Reply:
I looked that there is an equality check, so no problem if there was -1